src/Dist_Archive_Command.php

```diff
@@ -113,6 +116,25 @@ public function __invoke( $args, $assoc_args ) {
 			}
 		}
 
+		if ( isset( $assoc_args['plugin-dirname'] ) && rtrim( $assoc_args['plugin-dirname'], '/' ) !== $archive_base ) {
+			$plugin_dirname = rtrim( $assoc_args['plugin-dirname'], '/' );
+			$archive_base   = $plugin_dirname;
+			$tmp_dir        = sys_get_temp_dir() . DIRECTORY_SEPARATOR . $plugin_dirname . $version . '.' . time();
+			$new_path       = $tmp_dir . DIRECTORY_SEPARATOR . $plugin_dirname;
+			mkdir( $new_path, 0777, true );
+			// phpcs:ignore Squiz.PHP.DisallowMultipleAssignments.Found
+			foreach ( $iterator = new \RecursiveIteratorIterator( new \RecursiveDirectoryIterator( $path, \RecursiveDirectoryIterator::SKIP_DOTS ), \RecursiveIteratorIterator::SELF_FIRST ) as $item ) {
```

Comment:
Instead of suppressing linter warnings it could be written like this:
```suggestion
			$iterator = new \RecursiveIteratorIterator( new \RecursiveDirectoryIterator( $path, \RecursiveDirectoryIterator::SKIP_DOTS ), \RecursiveIteratorIterator::SELF_FIRST );
			foreach ( $iterator as $item ) {
```

Reply:
OK, I wasn't sure about that. I pulled that line from SO: https://stackoverflow.com/a/52976202/